RELEASES.md

```diff
@@ -7,6 +7,14 @@ Releases are proposed and adopted or rejected using the usual [project governanc
 An anti-pattern that we want to avoid is heavy development or discussions "late cycle" around major releases.
 We want to build a community that is involved and communicates consistently through all releases instead of relying on "silent periods" as a judge of stability.
 
+## Release Process
```

Comment:
The old `project.md` was runtime-spec-specific notes.  This `RELEASES.md` is [generic OCI policy][1].  I'd rather keep those in separate files.

[1]: https://github.com/opencontainers/project-template/pull/15